### ISSUE TITLE
vtk can be imported in console but it always report No module named 'vtk' in spyder

### ISSUE THREAD

**Issue by sdrgroup** [spyder-ide/spyder#3272]
## Description of your problem
I installed vtk7.0.0, and add followings in .bashrc (my OS is ubuntu16.04 64bit)
export LD_LIBRARY_PATH=/usr/local/VTK/lib:$LD_LIBRARY_PATH
export PYTHONPATH=/usr/local/VTK/lib:/usr/local/VTK/lib/python3.5/site-packages:/usr/local/VTK/bin:$PYTHONPATH 
I can run it correctly in python3 console:
import vtk
print(vtk.vtkVersion())

but when I run same script in spyder3, it always report :ImportError: No module named 'vtk'

I add same path into PYTHONPATH manager, it doesn't work. Can not spyder3 use the same configure with system?

## Versions and main components

* Spyder Version: sypder3 Spyder 2.3.8 
* Python Version: 3.5.1
* Operating system: ubuntu16.04

**Comment by jitseniesen** [comment]
Spyder should use the PYTHONPATH as set in the environment. Do you start it from a bash console? .bashrc is not run If a program is started from an icon or menu item.

When you add a path to the PYTHONPATH manager, that only takes effect in new Python or IPython consoles, not in existing ones.

You can print the path used to find modules for import with `import sys; print(sys.path)`

I hope these hints help you figure out what's going on.

**Comment by sdrgroup** [comment]
Thank you very much. When I started spyder3 in the console, it worked. I tried again if I start it from menu, spyder3 can not acquire the system's PYTHONPATH. 

When I installed spyder3 in the software centre, it made a short-cut in the menu automatically. So I often start spyder3 from menu. It takes me half a day to find a way to solve the problem, because I like to use spyder3 in python programming. When I decided to give up it, you help me solve the problem, I can use spyder3 again. 
Thank you.
